python/nx-cugraph/Makefile

```diff
@@ -16,9 +16,6 @@ lint-update:
 plugin-info:
 	python _nx_cugraph/__init__.py
 
-objects.inv:
-	wget https://networkx.org/documentation/stable/objects.inv
-
 .PHONY: readme
-readme: objects.inv
-	python scripts/update_readme.py README.md objects.inv
```

Comment:
@eriknw I removed the `objects.inv` download from the Makefile since it is no longer necessary, but I can't remember if you said you preferred to keep it or not.  The script will still accept an externally-provided `objects.inv` file specified as an arg, so restoring the Makefile will still work as intended.

Reply:
...and of course, running `make readme` will still work as expected.

Running `make readme` and the pre-commit hook will not download an `objects.inv` file if one already exists, so in both cases downloads only happen when necessary.

Reply:
I don't have a strong opinion. The nice thing about having a makefile target is it explicitly captures state and dependencies. Otherwise, this `objects.inv` will automagically appear when running `make`, which may be surprising.

Reply:
I restored `Makefile` in the latest commit.